CONTRIBUTING.md

```diff
@@ -227,24 +227,44 @@ To start debugging JHipster with **VSCode**, open the generator code in your wor
 
 It is also possible to debug sub generators by selecting one of the other debug options (for example `jhipster entity`). Those debug configurations are specified in the `.vscode/launch.json` file.
 
-## Local Travis Build
+## Local Travis Build and explore application generated
 
-You can run the travis builds locally by following below commands
+You can run the Travis builds locally by following below commands, and track rendered pages.
 
-CD into the travis folder `cd travis` from the generator source code root folder
+#### How to use it
 
-Run `./build-samples.sh [command_name] [sample_name:optional]`
+1. Change directory into the [./travis](./travis) folder `cd travis` from the generator source code root folder
 
-This will create the travis sample project under the `travis/samples` folder with folder name `[sample_name]-sample`. You can open this application in your editor or IDE to check it further. You can also run tests locally on the project to verify
+2. Do not forget to run `yarn link`.
 
-Sample name is optional and can be any of the folder name in the `travis/samples` folder. If not specified the it will mean all samples
+3. Run `./build-samples.sh generate|generateandtest [ sample_name[,sample_name][,...] ] [ --colorizelogfile ] ` (parameters between brackets are optionals).
 
-Command name can be as below
+    * `generate` and `generateandtest` create the Travis sample project under the `travis/samples` folder with folder name `[sample_name]-sample`. You can open this application in your Text Editor or IDE to check it further. You can also run tests locally on the project to verify.
+        * By default, run on all samples. Choose those you want in `./build-samples.sh help`.
+        * You could build several samples in same time by separated it by comma (e.g. `ngx-default,react-default`).
+        * `--colorizelogfile`: keep color in your log file, **this optional parameter is strongly advise for readability**. Check `./build-samples.sh help` to know more about it.
+    * `generateandtest` generate, and beside test samples like in Travis CI. Very interesting if you want check quickly know correctness of your modifications. Very interesting if you want check before push.
 
-    `list`: List all sample names
```

Comment:
please retain the list command, its useful

Reply:
TODO

Reply:
Done. But documented only in `./build-samples.sh help`. I believe it's cool to let only main command parameters in "Synopsis", and let newbies discover the others in `./build-samples.sh help`. Otherwise this section would be too long.